capsules/extra/src/net/network_capabilities.rs

```diff
@@ -49,7 +49,7 @@ impl AddrRange {
                 let full_bytes: usize = prefix_len / 8;
                 let remainder_bits: usize = prefix_len % 8;
                 // initial bytes -- TODO: edge case
-                if &allowed_addr.0[0..full_bytes] != &addr.0[0..full_bytes] {
+                if allowed_addr.0[0..full_bytes] != addr.0[0..full_bytes] {
```

Comment:
The fact that these two expressions are equivalent... this is why borrows are so hard to teach :/